modules/monitoring-creating-cluster-id-labels-for-metrics.adoc

```diff
@@ -3,98 +3,111 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="creating-cluster-id-labels-for-metrics_{context}"]
-= Creating cluster ID labels for metrics
 
-ifndef::openshift-dedicated,openshift-rosa[]
-You can create cluster ID labels for metrics for default platform monitoring and for user workload monitoring.
+// The ultimate solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
+// tag::CPM[]
+[id="creating-cluster-id-labels-for-metrics-cpm_{context}"]
+= Creating cluster ID labels for metrics for core platform monitoring
+// end::CPM[]
+
+// tag::UWM[]
+[id="creating-cluster-id-labels-for-metrics-uwm_{context}"]
+= Creating cluster ID labels for metrics for monitoring of user-defined projects
+// end::UWM[]
 
-For default platform monitoring, you add cluster ID labels for metrics in the `write_relabel` settings for remote write storage in the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace.
+// Set attributes to distinguish between cluster monitoring example (core platform monitoring - CPM) and user workload monitoring (UWM) examples
+// tag::CPM[]
+:configmap-name: cluster-monitoring-config
+:namespace-name: openshift-monitoring
+:component: prometheusK8s
+// end::CPM[]
+// tag::UWM[]
+:configmap-name: user-workload-monitoring-config
+:namespace-name: openshift-user-workload-monitoring
+:component: prometheus
+// end::UWM[]
 
-For user workload monitoring, you edit the settings in the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace.
+You can create cluster ID labels for metrics by adding the `write_relabel` settings for remote write storage in the `{configmap-name}` config map in the `{namespace-name}` namespace.
 
+ifndef::openshift-dedicated,openshift-rosa[]
+// tag::UWM[]
 [NOTE]
 ====
 When Prometheus scrapes user workload targets that expose a `namespace` label, the system stores this label as `exported_namespace`. 
 This behavior ensures that the final namespace label value is equal to the namespace of the target pod.
 You cannot override this default configuration by setting the value of the `honorLabels` field to `true` for `PodMonitor` or `ServiceMonitor` objects.
 ====
-
-endif::openshift-dedicated,openshift-rosa[]
-
-ifdef::openshift-dedicated,openshift-rosa[]
-You can create cluster ID labels for metrics by editing the settings in the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace.
```

Comment:
You said to ignore the ROSA/OSD "rendering." Just wanted to check if you intentionally removed this ifdef for OSD/ROSA

Reply:
Hello, yes, exactly. The ROSA/OSD ifdefs are used in most places because in those distros we only document monitoring for user-defined projects (UWM). With the new prepared changes, the UWM section will be separated from core platform monitoring, and thus these types of ifdefs will not be necessarry, because, those distros will only contain UWM tagged modules.